sentry/src/main/java/io/sentry/Hub.java

```diff
@@ -630,4 +632,79 @@ public void flush(long timeoutMillis) {
     }
     return clone;
   }
+
+  @ApiStatus.Internal
+  @Override
+  public SentryId captureTransaction(
```

Comment:
From what I can tell this is inlined into `Transaction.finish` in other SDKs. Can't tell if this is public API, but I would make it private to ensure the API people use is:

```
var t = Sentry.startTransaction();
t.finish();
```

not:

```
var t = Sentry.startTransaction();
Sentry.captureTransaction(t);
```

Reply:
It's not possible to inline it in Java. `captureTransaction` is marked as `ApiStatus.Internal` so I believe it's ok.

Reply:
even if it's not public, the reflection would make it possible to hack it anyway, of course not as straightforward as just calling a static method.